rpcpb/rpc.proto

```diff
@@ -300,6 +300,8 @@ message RestartNodeRequest {
   // Optional fields are set to the previous values if empty.
   optional string exec_path           = 2;
   optional string whitelisted_subnets = 3;
+
+  map<string, string> chain_configs = 4;
```

Comment:
I am a bit confused, hadn't this already been done @felipemadero

Reply:
this is supported on node start but not sure how it works if you want to restart a node with a different config.

Reply:
Ohh the param is absent from restart node, yes, this should be here. Will also do a review of all of this by tomorrow.